appwindow.js

```diff
@@ -107,6 +107,26 @@ var AppWindow = GObject.registerClass(
             this.method_handler(this.settings, 'changed::window-position', this.update_resize_boxes);
             this.update_resize_boxes();
 
+            const HEIGHT_MOD = 0.05;
+            this.simple_action('window-inc-height', () => {
+                this.adjust_double_setting('window-size', HEIGHT_MOD);
+            });
+            this.simple_action('window-dec-height', () => {
+                const current = this.settings.get_double('window-size');
+                if (current < HEIGHT_MOD)
+                    this.hide();
```

Comment:
Don't do this. The hotkey is for decreasing window size, not hiding the window. `window-size = 0` is fine; you can set it to `0` in the preferences dialog. The window will just have the smallest size possible.

Reply:
Got it!